packages/salesforcedx-vscode-core/src/commands/util/sfdxCommandlet.ts

```diff
@@ -21,6 +21,7 @@ import { channelService } from '../../channels';
 import { notificationService, ProgressNotification } from '../../notifications';
 import { taskViewService } from '../../statuses';
 import { TelemetryData, telemetryService } from '../../telemetry';
+import { Measurements, Properties } from '../../telemetry/telemetry';
```

Comment:
Do we want to expose Measurements/Properties in `index.ts` of the telemetry module?